app/upgrades_test.go

```diff
@@ -1042,8 +607,7 @@ func (s *UpgradeTestSuite) TestViridian() {
 	s.AssertUpgradeHandlerLogs("viridian", expInLog, nil)
 }
 
-func (s *UpgradeTestSuite) TestMetadataMigration() {
-	// TODO: Delete this test with the rest of the viridian stuff.
+func (s *UpgradeTestSuite) TestMetadataMigration() { // TODO[viridian]: Delete this test after the upgrade.
```

Comment:
No. That's ridiculous. There are no automated processes in any of our stuff that would try to handle something like that. And I don't want an automated script ripping out large chunks of code, specially unit tests.

The `TODO[viridian]` on it is all we need. I'm certain that it will be properly deleted when the time comes.